src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -0,0 +1,154 @@
+		<SingleProject>true</SingleProject>
+		<OutputType>Exe</OutputType>
+		<IsUnoHead>true</IsUnoHead>
+		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
+		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">26.0</SupportedOSPlatformVersion>
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Uno.WinUI" Version="4.6.19" />
+		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.6.19" Condition="'$(Configuration)'=='Debug'" />
+		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.6.19" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
+		<PackageReference Include="Uno.WinUI.Lottie" Version="4.6.19" />
+	</ItemGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\ApplicationTemplate.Business\ApplicationTemplate.Business.csproj" />
+		<ProjectReference Include="..\ApplicationTemplate.Client\ApplicationTemplate.Client.csproj" />
+		<ProjectReference Include="..\ApplicationTemplate.Presentation\ApplicationTemplate.Presentation.csproj" />
+	</ItemGroup>
+	<Choose>
+		<When Condition="'$(TargetFramework)'=='net6.0-android'">
+			<Choose>
+				<When Condition="'$(Configuration)'=='Debug'">
+					<PropertyGroup>
+						<AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
+						<UseHighDPIResources>False</UseHighDPIResources>
+						<EmbedAssembliesIntoApk>False</EmbedAssembliesIntoApk>
+						<AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
+						<AotAssemblies>False</AotAssemblies>
+						<EnableLLVM>False</EnableLLVM>
+						<AndroidEnableProfiledAot>False</AndroidEnableProfiledAot>
+						<BundleAssemblies>False</BundleAssemblies>
+						<AndroidDexTool>d8</AndroidDexTool>
+					</PropertyGroup>
+				</When>
+				<When Condition="'$(Configuration)'=='Release'">
+					<PropertyGroup>
+						<EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
+						<AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
+						<AndroidManagedSymbols>True</AndroidManagedSymbols>
+						<AndroidEnableSGenConcurrent>True</AndroidEnableSGenConcurrent>
+						<AndroidAotAdditionalArguments>nodebug,no-write-symbols</AndroidAotAdditionalArguments>
+						<AndroidUseDefaultAotProfile>False</AndroidUseDefaultAotProfile>
+						<BundleAssemblies>False</BundleAssemblies>
+						<AndroidUseAapt2>True</AndroidUseAapt2>
+					</PropertyGroup>
+					<Choose>
+						<When Condition="'$(IsLightBuild)'=='True'">
+							<!-- Build Optimization for LightBuild (Faster Builds in Pull Requests). -->
+							<PropertyGroup>
+								<AndroidEnableProfiledAot>False</AndroidEnableProfiledAot>
+								<EnableLLVM>False</EnableLLVM>
+								<Optimize>False</Optimize>
+								<RunAOTCompilation>False</RunAOTCompilation>
+							</PropertyGroup>
+						</When>
+						<Otherwise>
+							<PropertyGroup>
+								<!-- Currently doesn't work with AOT enabled. -->
```

Comment:
Do we have a trace here for that? (should probably have a github issue in this repo as I don't think it's a good idea to reference anything private here)

Reply:
No nothing and I can't even have a log for this, using release mode and too much text in the logcat.

Reply:
yeah that's perfect